### DISCUSSION TITLE
How to perform Peak2Gene analysis using ATAC and RNA from same cells?

### DISCUSSION THREAD

**Discussion by lixining2020** [GreenleafLab/ArchR#1091] [Questions / Documentation]
Hi, thanks for the great tools! I am not sure how can I perform  peak2gene analysis when I have ATAC data and RNA data from the exactly same cells by 10x platforms. It seems that addGeneIntegrationMatrix function is not tailored for adding scRNA info to the existing scATAC data. Thanks in advance for any suggestion!

**Comment by rcorces** [comment]
We are working on incorporating this into the manual. For now, it is part of a separate multiome tutorial
https://greenleaflab.github.io/ArchR_2020/Ex-Analyze-Multiome.html

**Reply by lixining2020** [reply]
Thank you so much for the very quick replies, and I also found this https://github.com/GreenleafLab/ArchR/issues/507 is really helpful and hope to see the improvements in the latter version!
